cmd/utils/flags.go

```diff
@@ -156,6 +153,11 @@ var (
 		Name:  "lightkdf",
 		Usage: "Reduce KDF memory & CPU usage at some expense of KDF strength",
 	}
+	EthModeFlag = cli.StringFlag{
+		Name:  "mode",
+		Value: "archive",
+		Usage: "Client mode of operation (archive, full, light)",
+	}
```

Comment:
This flag was dropped because it's too murky what it represents. Please rework the code so that it is based on a single `--light` flag. If it is passed, the `cmd/geth/` package should initialize `eth` and `les` properly.